da/da.go

```diff
@@ -221,6 +227,9 @@ type DataAvailabilityLayerClient interface {
 
 	// Returns the maximum allowed blob size in the DA, used to check the max batch size configured
 	GetMaxBlobSizeBytes() uint32
+
+	// GetSignerBalance returns the balance for a specific address
+	GetSignerBalance() (*Balance, error)
```

Comment:
is there any reason to return pointer to `Balance` here instead of the value?

Reply:
it doesn't! this is our interface that we do control, i believe you'll just need to remove a pointer from respective method. let's change to a value receiver then to avoid panics and confusion while reading the code.

Reply:
i dont think its a big issue, but i think it can be dealt inside our wrapper func to the celestia openrpc call and directly return the value in GetSignerBalance